stable/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -72,6 +72,7 @@ spec:
                 {{- if .Values.nginx.customArtifactoryConfigMap }}
                 cp -Lrf /tmp/conf.d/artifactory*.conf /etc/nginx/conf.d/artifactory.conf;
                 {{- else }}
+                until [ -f /etc/nginx/conf.d/artifactory.conf ]; do echo ; done;
```

Comment:
Please add a `sleep 1` in the loop so CPU will not spike.

Reply:
Done